ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/UtxoLedgerService.java

```diff
@@ -271,4 +272,30 @@ void sendUpdatedTransactionBuilder(
      */
     @Suspendable
     @NotNull <R> VaultNamedParameterizedQuery<R> query(@NotNull String queryName, @NotNull Class<R> resultClass);
+
+    /**
+     * Persists a non-finalized (draft) transaction.
+     * It executes the same checks as the finalization does before its initial persist.
+     * So the transaction and all of its signatures need to be valid, but there may be missing signatures.
+     * <p>
+     * @param utxoSignedTransaction The {@link UtxoSignedTransaction} to persist.
+     * @throws ContractVerificationException if the transaction fails contract verification.
+     * @throws TransactionSignatureException if a signature of the transaction fails verification.
+     * @throws IllegalStateException if the transaction does not have any signatures.
+     */
+    @Suspendable
+    void persistDraftSignedTransaction(
```

Comment:
`draft transaction` is term mostly used within the interop team, not sure if the wider audience will understand it (platfrom), alternatively it could be (maybe ugly name) `persistPartiallySignedTransaction`, but for me it's fine